docs/README.fedora

```diff
@@ -0,0 +1,78 @@
+TOC
+1. Introduction
+2. Getting the source code
+3. Installing the required libraries and headers
+4. How to compile
+5. Uninstalling
+
+-----------------------------------------------------------------------------
+1. Introduction
+-----------------------------------------------------------------------------
+
+A graphics-adapter with OpenGL or OpenGLES acceleration is required.
+
+Note to new Linux users:
+All lines that are prefixed with the '$' character are commands,
+that need to be typed into a terminal window / console. The '$' equals the prompt.
+Note: The '$' character itself should NOT be typed as part of the command.
+
+Note to new Fedora users:
+For downloading packages use, $ sudo dnf <package_name>
+
+-----------------------------------------------------------------------------
+2. Getting the source code
+-----------------------------------------------------------------------------
+
+You will have to grab the source code of course, here we use git as example.
+First install the git package provided by your distribution.
+Then from a terminal, type:
+
+.0  $ cd $HOME
+.1  $ git clone git://github.com/xbmc/xbmc.git kodi
+
+Note: You can clone any specific branch.
+
+.1  $ git clone -b <branch> git://github.com/xbmc/xbmc.git kodi
+
+-----------------------------------------------------------------------------
+3. Installing the required libraries and headers
+-----------------------------------------------------------------------------
+
+You will then need the required libraries. The following is the list of packages
+that are used to build Kodi packages on FEDORA (with all supported
+external libraries enabled).
+
+Build-Depends: sudo dnf autoconf automake gettext cmake curl
```

Comment:
You should format this as command (so it should start with `$`) and allow users to copy-paste it into their terminal, so it would be better to include line continuations (`\`) at the end of each line. Also there is no need to write `Build-Depends` at the beginning.

Reply:
Also, should be `dnf install`

Reply:
Done. I guess I finished all the required changes. Please review and let me know if you have any comments.